lib/sedna/algorithms/aggregation/mistnet.py

```diff
@@ -0,0 +1,24 @@
+from plato.servers import mistnet
+
+from .base import BaseAggregation
+
+
+class MistNet(BaseAggregation):
```

Comment:
If it needs to be called by the registration factory, maybe we can add the registration wrap with `ClassFactory`.

Reply:
Jie Pu has updated it.